### PR TITLE
feat: added product listing page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created Product Listing page containing filters by:
1. Price
2. Category
3. Ratings
4. Sort By

index-1
index-2
index-3

### CODE REVIEW

File: pages/products.html [hunk 1]
```diff
@@ -1 +1,328 @@
...
 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/products.css">
```
Comment (krishanu2911) [inline pages/products.html]: just a small suggestion like you can import your CL link the base CSS file and import only one base or main CSS file into all your HTML header files will easier for you only have link to add in Html

**Review by Sameony — APPROVED** [review]
Great use of component library classes. Nice code

File: stylesheets/products.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    background-color: var(--gray-200);
+}
+
+.slider-input {
+    width: 12rem;
+}
+
+.price-range {
+    width: 12rem;
+}
```
Comment (sadathshariff) [inline stylesheets/products.css]: Just a suggestion it's repeating over here since it's just width can be added together

**Review by krishanu2911 — APPROVED** [review]
Nice design and well-written code nice usage of class name everything is good to go no changes required, just added a small suggestion

**Review by sadathshariff — APPROVED** [review]
The code looks clean and good use of utility classes

File: pages/products.html (hunk 1 above)
Comment (nakulsharma15) [reply pages/products.html:12]: Ok, I'll do it. Thanks for Pointing out.

Comment (nakulsharma15) [reply pages/products.html]: just did in the latest commit. Thanks for the suggestion.

File: stylesheets/products.css (hunk 2 above)
Comment (nakulsharma15) [reply stylesheets/products.css]: Just did in the latest commit. Thanks for pointing out.

**Pull request merged**